index.js

```diff
@@ -64,6 +64,19 @@ function installLibbpf(libbpf_version) {
 function installKubectl(kubectl_version) {
   core.info(`Get kubectl with version `+ kubectl_version);
   executeCommand("curl -LO https://dl.k8s.io/release/v"+kubectl_version+"/bin/linux/amd64/kubectl", "fail to install kubectl");
+  executeCommand("chmod +x ./kubectl");
+  executeCommand("mv ./kubectl /usr/local/bin/kubectl");
+  core.info(`Get kustomize`);
+  executeCommand("curl -s \"https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh\" | bash");
+  executeCommand("chmod +x kustomize");
+  executeCommand("mv kustomize /usr/local/bin/");
+}
+
+function installKind() {
+  core.info(`Get Kind with latest version`);
+  executeCommand("curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.22.0/kind-linux-amd64","fail to install kind");
```

Comment:
Do we want to make the Kind version fixed? Can we make it something like `installKind(kind_version)`?

Reply:
let's make the version been fixed for now, we can improve in PR's next.

Reply:
#107 openned.